nasa/openmct
# [Time conductor] Better persistence handling for history

## Overview
Addresses #3189 

## Expected Behavior
- Changing any bounds should save a record in the time conductor history dropdown menu (located below time conductor axis)
- Clicking on any record should change bounds to those saved in record
- History should persist across sessions

## Testing Instructions
- ensure saving records happens instantly, on first load, and does not require reloading of page to sync
- ideally should be tested on openmct deployment (where original issue was seen) or vista deployment vs local dev machine
- can delete local storage record to retest


## Author Checklist
| | |
| --- | :---: |
| Changes address original issue? | Y |
| Unit tests included and/or updated with changes? | N/A (created #3244) |
| Command line build passes? | Y |
| Changes have been smoke-tested? | Y |
| Testing instructions included? | Y |
